src/version_checker.cpp

```diff
@@ -32,9 +33,9 @@ Version parseLatestReleasedVersion(QString data)
 {
     Version newest;
     for (auto && line : data.split("\n")) {
-        QRegExp tagMatch("^*tag/(\\d.\\d.\\d).*$");
-        if (const auto index = tagMatch.indexIn(line); index != -1) {
-            const Version version(tagMatch.cap(1));
+        QRegularExpression tagMatch("\\d{1,3}.\\d.\\d?");
```

Comment:
How can you be sure this matches strings like "releases/tag/3.6.0" when you remove "^*tag/" ?